addons-cluster/zookeeper/values.yaml

```diff
@@ -2,6 +2,10 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+## @param version cluster version
+##
+version: 3.9.2
```

Comment:
better to rename to `serviceVersion`

Reply:
renamed

Reply:
apecloud use .Values.version  to decide which version to use. Better to keep this name